deploy/prepare_deployment.py

```diff
@@ -184,6 +184,12 @@
     './win_build/Build/ARM64/Release/boincmgr.exe'
 ]
 
+wasm_client_list = [
+    './client/boinc_client.wasm',
+    './client/boinc',
```

Comment:
Is this a JS client? If yes - it should contain `.js` extension to not mix it with linux binary

Reply:
The boin is JavaScript. The compiler create it boinc without JavaScript because config.ac file. 
Not sure if I succeeded to rename it.

Reply:
I'm sure you can: https://github.com/BOINC/boinc/blob/master/configure.ac#L375
Just need to find where to set this variable properly